content/Proposals/Accepted/202205-oo.md

```diff
@@ -0,0 +1,63 @@
+- Resource-constraint deployments need a stripped down version of the monitoring stack, e.g. only forwarding signals to a central infrastructure or disabling certain parts of the stack completely.
+- Mixed deployments (e.g. OpenShift + OpenStack) are conceptually very similar to the multi-cluster use case, also needing a central pane of glass for observability signals.
+- Special purpose deployments (e.g. OpenShift Data Foundation) have special requirements when it comes to monitoring, that are tricky to align with the existing CMO setup.
+- Looking at eventually correlating different observability signals also the cluster-side stack would potentially benefit from a holistic approach for deploying monitoring, logging and tracing components and configuring them in the right way to work together.
+- Managed service deployments need a multi tenancy capable way of deploying many similarly built monitoring stacks to a single cluster. This is the short-term focus for OO.
+
+The proposal is to combine all these (and more) use cases into one single (meta) operator (as recommended by [the operator best practices](https://sdk.operatorframework.io/docs/best- practices/best-practices/#development)) which can be configured with e.g. presets to instruct lower-level operators (like prometheus-operator, potentially Loki operator or Jaeger one) to deploy purpose-built monitoring stacks for different uses cases. This is similar to the original CMO concept but with much higher flexibility, and feature velocity in mind, thanks to not being tied to OpenShift Core versioning.
+
+Additionally, supporting multiple different ways of deploying monitoring stacks (CMO as the standard OpenShift way, OO for managed services, something else for e.g. HyperShift or edge scenarios, ...) is a burden for the team. Instead, eventually supporting only one way for deploy monitoring stacks - with OO - covering all these uses cases makes it a lot simpler and far more consistent.
+
+### Pitfalls of the current solution
+
+CMO is built for traditional self-operated single-cluster focused deployments of OpenShift. It intentionally lacks the flexibility for many other use cases (see above) in order to provide monitoring that is resilient against configuration drift. E.g. the reason for creating OO (MSO) in the first place - supporting managed service uses cases - can't currently be covered by CMO. See the [original MSO proposal](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/monitoring-stack-operator.md) for more details.
+
+The results of this lack of flexibility can be readily observed: Red Hat teams have built their own solutions for their monitoring use cases, e.g. leveraging community operators or self-written deployments, with varying success, reliability and supportability.
+
+## Goals
```

Comment:
Is it worth stating explicitly that OO focuses on edge locations (e.g. it will collect, process and forward observability signals but it isn't in the business of providing a highly-scalable observability store)?

Reply:
I think this is already covered in the "resource constraint deployments" use case above.

Reply:
But then we should also mention all other use cases, shouldn't we? Why is edge more important (and thus worth to point out explicity) than e.g. multi-cluster or correlation use cases?

Reply:
Not feeling strongly about this one. The non-goals section states that OO won't deal with multi-cluster deployments so I'm fine :)